test/Replay/libkleeruntest/replay_detection.c

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang %s -emit-llvm -g %O0opt -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out --search=dfs %t.bc
```

Comment:
I would remove `--search=dfs`; it's better to only add the options that are needed.

Reply:
Done